angular/projects/admin-nrpti/src/app/mines/mines-collection-detail/mines-collection-detail-record-row/mines-collection-detail-record-row.component.html

```diff
@@ -0,0 +1,14 @@
+<td scope="row" data-label="Name" class="col-6">{{ rowData.name || '-' }}</td>
+<td scope="row" data-label="Source System" class="col-2">{{ rowData._sourceSystem || '-' }}</td>
+<td scope="row" data-label="Date" class="col-2">{{ formatDate(rowData.date) || '-' }}</td>
```

Comment:
Might need a `rowData.date && formatDate(..)` null check.  Otherwise moment might spit out some default 1970 date or something, if date is null.

Reply:
Fixed.